hardening/oscap/old-new/test.py

```diff
@@ -22,13 +22,13 @@
     oscap.unselect_rules(util.get_datastream(), 'remediation-new.xml', remediation.excludes())
     g.copy_to('remediation-new.xml')
 
-    def remediate(datastream, html_report, html_report2):
+    def remediate(datastream, arf_results, arf_results2):
         # remediate twice due to some rules being 'notapplicable'
         # on the first pass
-        for html_report in [html_report, html_report2]:
+        for results in [arf_results, arf_results2]:
```

Comment:
Based on flake8, we should rather pick some different variable name here to not shadow the imported module.